src/tests/functional/form_mode_tests.ts

```diff
@@ -65,11 +65,11 @@ test("test form submission with form mode optin and form enabled from submitter
   assert.ok(await formSubmitStarted(page))
 })
 
-async function gotoPageWithFormMode(page, formMode) {
+async function gotoPageWithFormMode(page: Page, formMode: "on" | "off" | "optin") {
```

Comment:
Would be great to leave this as just `formMode: string`. If you want to contribute to this project you should put in the work figuring out what that actual options are yourself.